src/screens/Sleep.tsx

```diff
@@ -1,9 +1,144 @@
 import React from 'react';
-import { View } from 'react-native';
+import {View, Text, StyleSheet, TouchableOpacity, Alert,} from 'react-native';
```

Comment:
You guys have random semi-colons throughout your code. I think that you should delete all of them because they are not needed and your code will look a lot cleaner.

```suggestion
import {View, Text, StyleSheet, TouchableOpacity, Alert,} from 'react-native'
```